src/main/java/edu/kis/powp/jobs2d/drivers/canva/WorkspaceDriver.java

```diff
@@ -0,0 +1,59 @@
+package edu.kis.powp.jobs2d.drivers.canva;
+
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.jobs2d.drivers.canva.shapes.CanvaShape;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+
+public class WorkspaceDriver implements Job2dDriver {
+    private final Job2dDriver innerDriver;
+
+    private final Job2dDriver borderDriver;
+
+    private int currentX,  currentY;
+    private CanvaShape bound;
```

Comment:
Again: Adding "Workspace feature" it is important to provide access to current canvas setting to the whole application so other features can take advantege of it. 
I think it could something like get(/set)CurrentDriver -> get(/set)CurrentCanva but in WorkspaceFeature.